app/routes/project/settings/donations.js

```diff
@@ -0,0 +1,23 @@
+import Ember from 'ember';
+
+const {
+  inject: { service },
+  Route
+} = Ember;
+
+export default Route.extend({
+  ajax: service(),
+
+  model() {
+    let project = this.modelFor('project');
+    return project;
+  },
+
+  afterModel(model) {
+    this.store.queryRecord('stripe-auth', {
```

Comment:
When working on the ember-guides repo, I remember them saying that `this.get('store')` was the preferred convention going forward. I can't remember the exact reason, but the tutorials are being updated to use that syntax.